Sources/KituraContracts/CodableQuery/QueryEncoder.swift

```diff
@@ -199,7 +211,215 @@ public class QueryEncoder: Coder, Encoder, BodyEncoder {
+        case let fieldValue as UInt32:
+            encoder.dictionary[fieldName] = String(fieldValue)
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Int Arrays
+        case let fieldValue as UInt64:
+            encoder.dictionary[fieldName] = String(fieldValue)
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Int Arrays
+        /// UInt Arrays
+        case let fieldValue as [UInt]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Int Arrays
+        case let fieldValue as [UInt8]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        case let fieldValue as [UInt16]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        case let fieldValue as [UInt32]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        case let fieldValue as [UInt64]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Floats
+        case let fieldValue as Float:
+            encoder.dictionary[fieldName] = String(fieldValue)
+            encoder.anyDictionary[fieldName] = fieldValue
+        case let fieldValue as [Float]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Doubles
+        case let fieldValue as Double:
+            encoder.dictionary[fieldName] = String(fieldValue)
+            encoder.anyDictionary[fieldName] = fieldValue
+        case let fieldValue as [Double]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Bools
+        case let fieldValue as Bool:
+            encoder.dictionary[fieldName] = String(fieldValue)
+            encoder.anyDictionary[fieldName] = fieldValue
+        case let fieldValue as [Bool]:
+            let strs: [String] = fieldValue.map { String($0) }
+            encoder.dictionary[fieldName] = strs.joined(separator: ",")
+            encoder.anyDictionary[fieldName] = fieldValue
+        /// Strings
+        case let fieldValue as String:
+            encoder.dictionary[fieldName] =+= fieldValue
```

Comment:
Is this the only case where we need to build a comma-separated list?  What if a custom date encoding serializes to `Int` or `Double` instead?